exporters/sender/okhttp/src/test/java/io/opentelemetry/exporter/sender/okhttp/internal/OkHttpGrpcSenderTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.sender.okhttp.internal;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.opentelemetry.exporter.internal.RetryUtil;
+import io.opentelemetry.exporter.internal.grpc.GrpcExporterUtil;
+import java.util.Set;
+import okhttp3.MediaType;
+import okhttp3.Protocol;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class OkHttpGrpcSenderTest {
```

Comment:
#nit: we omit `public` on test classes and test methods in this repo, since junit 5 doesn't require them to be public
```suggestion
class OkHttpGrpcSenderTest {
```

Reply:
Thanks for the code review😊